.github/workflows/reusable-build-push.yml

```diff
@@ -0,0 +1,198 @@
+name: Reusable workflow for build and push multi-arch images
+
+# Run workflow by manual dispatch
+on:
+  workflow_call:
```

Comment:
You say manual dispatch in the comment but it's using the `workflow_call` event, which is triggered only by other workflows. Is the comment wrong or is the event wrong?

Reply:
Yes the comment was wrong